src/Tokenizer/TokensAnalyzer.php

```diff
@@ -525,33 +536,32 @@ public function isWhilePartOfDoWhile($index)
     }
 
     /**
-     * @param Tokens $tokens
-     * @param int    $start
+     * Find classy elements.
      *
-     * @return array
+     * Searches in tokens from the classy index till the end of the classy.
+     * Returns an 2D array; (int) the index until the method has analysed, (array) found classy elements.
```

Comment:
I would write _"Returns an array with last analyzed indexes as keys and arrays of found tokens as values."_

Reply:
it only returns one index (the last analyzed) and the elements found

Reply:
Ah you're right, then I would write: _"Returns an array with the classy end index as first value and an array of found tokens as second value."_